backend/app/donation/routes.py

```diff
@@ -1,6 +1,101 @@
 from flask import request, abort, jsonify
-from app.models import Donation
 from . import donation
+from .. import db
+from app.models import Donation
+import uuid
+from datetime import datetime
+
+# Create new donation
+@donation.route("", methods=["POST"])
+def create_donation():
+    # Load request to data
+    data = request.get_json(force=True)
+    # Assign id new uuid, and set date to now
+    id = uuid.uuid4()
```

Comment:
@ahmad675 it's fine to not create an id, as it is automatically created once it is inserted.